.github/workflows/test.yml

```
@@ -5,12 +5,21 @@ name: Test
 on:
   push:
     branches: [ master ]
+    paths:
+      - '**.gemfile'
+      - '**.rb'
+      - '.github/workflows/**'
+      - '.github/actions/**'
+      - 'Gemfile*'
+      - '!bin/**'
   pull_request:
     type: [ opened, synchronize, reopened, review_requested ]
```

actionlint (1.7.12)

[error] unexpected key "type" for "pull_request" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)